.github/workflows/branch-to-staging.yml

```diff
@@ -15,6 +15,13 @@ on:
         type: string
 
 jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Display parameters
+        shell: bash
+        run: echo "${{ format('<h1>Parameters</h1><p>{0}</p>', tojson(inputs)) }}" > "$GITHUB_STEP_SUMMARY"
```

Comment:
Logs the parameters we run the job with, useful when we select only a few and need to debug which ones use used